will/backends/encryption/aes.py

```diff
@@ -1,65 +1,67 @@
+'Encrypt stored data'
+
 import binascii
-import base64
-import codecs
-import dill as pickle
 import hashlib
 import logging
-from Crypto.Cipher import AES
-import random
 import os
-import traceback
+
+import dill as pickle
+from Crypto.Cipher import AES
 
 from will import settings
 from will.backends.encryption.base import WillBaseEncryptionBackend
 
+# pylint: disable=no-member
 
 BS = 16
 key = hashlib.sha256(settings.SECRET_KEY.encode("utf-8")).digest()
 
 
-def pad(s):
-    s = "%s%s" % (s.decode("utf-8"), ((BS - len(s) % BS) * "~"))
-    return s
+def pad(s: bytes) -> str:
+    '''Ensure the data to be encrypted has sufficient padding.
+    Arbitrarily adding ~ to the end, so your message better not end with ~.'''
+    return "%s%s" % (s.decode("utf-8"), ((BS - len(s) % BS) * "~"))
 
 
-def unpad(s):
-    while s.endswith(str.encode("~")):
-        s = s[:-1]
-    return s
+def unpad(s: bytes) -> bytes:
+    'Removes all ~ on the end of the message.'
+    return s.rstrip(b'~')
 
 
 class AESEncryption(WillBaseEncryptionBackend):
-
-    @classmethod
-    def encrypt_to_b64(cls, raw):
+    'AES encryption backend'
+    @staticmethod
+    def encrypt_to_b64(raw):
+        'encrypt and b64-encode data'
         try:
             enc = binascii.b2a_base64(pickle.dumps(raw, -1))
             if settings.ENABLE_INTERNAL_ENCRYPTION:
                 iv = binascii.b2a_hex(os.urandom(8))
                 cipher = AES.new(key, AES.MODE_CBC, iv)
                 enc = binascii.b2a_base64(cipher.encrypt(pad(enc)))
                 return "%s/%s" % (iv.decode("utf-8"), enc.decode("utf-8"))
-            else:
-                return enc
-        except:
-            logging.critical("Error preparing message for the wire: \n%s" % traceback.format_exc())
+            return enc
+        except Exception:
+            logging.exception("Error preparing message for the wire: \n%s", raw)
             return None
 
-    @classmethod
-    def decrypt_from_b64(cls, raw_enc):
+    @staticmethod
+    def decrypt_from_b64(enc):
+        'decrypt b64-encoded data'
         try:
```

Comment:
can you explain the change to the `decrypt_from_b64` method?

Reply:
The original declaration didn't actually reference the class in any way.